lib/packages.cf

```diff
@@ -94,6 +94,28 @@ body package_module yum
     #default_options => {};
 }
 
+body package_module freebsd_ports
+# @brief Define details used when interfacing with the freebsd ports package
+# module.
+#
+# **Note:** Ports are expected to be setup prior to trying to use the packages
+# promise. You may need to ensure that `portsnap extract` has been run.
+#
+# **Example:**
+# ```cf3
+# bundle agent main
+# {
+#   packages:
+#     freebsd::
```

Comment:
I tried out the apt_get package module on FreeBSD and it didn't fail very gracefully. That leaves me less inclined to handle that scenario... if someone tries to use FreeBSD ports on Debian, that's their lookout :P